base/cvd/cuttlefish/host/commands/cvd/unittests/selector/instance_database_test.cpp

```diff
@@ -48,13 +48,21 @@
 namespace cuttlefish {
 namespace selector {
 
+#define CF_ASSERT_OK(c)     \
+  ({                        \
+    decltype(c)&& nc = c;   \
+    ASSERT_TRUE((nc).ok()); \
+    *(nc);                  \
+  })
```

Comment:
Can this use the helpers in [`result_matchers.h`](https://github.com/google/android-cuttlefish/blob/315c7310d97c6f7fbccbdd9a5c46f7370be4761f/base/cvd/cuttlefish/common/libs/utils/result_matchers.h#L27)?

Reply:
Done